pkgs/applications/networking/browsers/vivaldi/default.nix

```diff
@@ -10,23 +10,23 @@
 }:
 
 let
-  archUrl = name: arch: "https://vivaldi.com/download/stable/${name}_${arch}.deb";
+  archUrl = name: arch: "https://downloads.vivaldi.com/stable/${name}_${arch}.deb";
 in
 stdenv.mkDerivation rec {
-  version    = "1.0";
-  debversion = "stable_1.0.435.40-1";
+  version    = "1.1";
+  debversion = "stable_1.1.453.47-1";
   product    = "vivaldi";
   name       = "${product}-${version}";
 
   src = if stdenv.system == "x86_64-linux"
```

Comment:
Also, I believe `meta.platforms` makes throwing strictly unnecessary, as you'll get an assertion failure anyway, trying to build it on an unsupported platform.